views/check-your-answers.html

```diff
@@ -59,12 +59,23 @@ <h1 class="govuk-heading-xl">Check your answers before sending your application<
 
       <br>
       <form method="post">
-        {{ govukButton({
-          text: "Continue to payment",
-          attributes: {
-            "id": "submit"
-          }
-        }) }}
+        <div class="govuk-button-group">
+          {{ govukButton({
+            text: "Continue to payment",
+            attributes: {
+              "id": "submit"
+            }
+          }) }}
+
+          {{ govukButton({
+            text: "Print this page",
+            classes: "govuk-button--secondary",
```

Comment:
should also have a unit test to check that the print button is present, not sure how to check if not present when JS disabled though.